src/const.ts

```diff
@@ -11,3 +11,26 @@ export interface Manifest {
   name: string;
   builds: Build[];
 }
+
+export interface FlashState {
+  state: State;
+  message: string;
+  details?: Record<string, any>;
+}
+
+export enum State {
```

Comment:
Once `FlashState` is a union, this can be come `type State = FlashState["state"]`